File: FileRenamer/ViewModels/Controls/About/ThanksViewModel.cpp
#include "pch.h"
#include "ThanksViewModel.h"
#include "ThanksViewModel.g.cpp"

namespace winrt::FileRenamer::implementation
{
	ThanksViewModel::ThanksViewModel()
	{
		_thanksDict.Append(winrt::make<ThanksKeyValuePairModel>(L"MouriNaruto", L"https://github.com/MouriNaruto"));
	}

	winrt::WinrtCollections::IObservableVector<winrt::FileRenamer::ThanksKeyValuePairModel> ThanksViewModel::ThanksDict()
	{
		return _thanksDict;
	}
}

File: FileRenamer/ViewModels/Controls/About/ThanksViewModel.cs
using FileRenamer.Models.Controls.About;
using System.Collections.Generic;

namespace FileRenamer.ViewModels.Controls.About
{
    public sealed class ThanksViewModel
    {
        public List<ThanksKeyValuePairModel> ThanksDict = new List<ThanksKeyValuePairModel>()
        {
            new ThanksKeyValuePairModel(){Key = "MouriNaruto" ,Value = "https://github.com/MouriNaruto" },
        };
    }
}
